use-timescale/continuous-aggregates/compression-on-continuous-aggregates.md

```diff
@@ -12,19 +12,10 @@ is only used for analytical queries and never modified, you can compress the
 aggregate to save on storage.
 
 Compression on continuous aggregates works similarly to
-[compression on hypertables][compression]. Compressed chunks can be queried, but
-they can't be updated or deleted. For continuous aggregates, that means
-compressed chunks can't be refreshed.
-
-If you receive historical data and need to refresh a compressed chunk, see
-[the troubleshooting guide][troubleshooting].
-
-<Highlight type="warning">
-You can't refresh the compressed regions of a continuous aggregate. To avoid
-conflicts between compression and refresh, make sure you set `compress_after`
-to a larger interval than the `start_offset` of your
-[refresh policy](/api/latest/continuous-aggregates/add_continuous_aggregate_policy).
-</Highlight>
```

Comment:
What about instead of removing the warning change the message mentioned since which version is possible to use refresh?

Reply:
Added the warning for old versions.